### DISCUSSION TITLE
Find/Till with multiple characters

### DISCUSSION THREAD

**Discussion by chookity-pokk** [meow-edit/meow#398] [Ideas]
I have tried to get into the habit of using find but I'm almost never that close to the one character you're allowed to input. If it could take two characters I think it'd become much more useable on a regular basis. It could be bound to capital F so that way f can stay as find.

**Comment by DogLooksGood** [comment]
Does the digit hints work for you, so after `f` you can press 0-9 to quickly expand the selection.

**Reply by chookity-pokk** [reply]
It works, but I find myself in the situation where I have to hit multiple numbers is a row to get it to where I want. A simple example is my elfeed config (below). If I'm at the top and want to jump down to my key binding for elfeed there's about 54 e's between them so even with the 0-9 it isn't exactly super usable. 


```
(setq elfeed-feeds
      '(("https://ooohspooky.libsyn.com/rss" AdamKnox)
	("https://godotengine.org/rss.xml" Godot)
	("https://stackabuse.com/rss/" StackAbuse)
	("https://opensource.com/feed" OpenSource)
	("https://daverupert.com/atom.xml" DaveRupert)
	("https://feed.podbean.com/mateitsasurething/feed.xml" Wolfe)
	("https://www.omnycontent.com/d/playlist/e73c998e-6e60-432f-8610-ae210140c5b1/0c6fc944-eaa5-4022-8e77-ae3300346d76/45a735b6-e1b8-49f9-a595-ae3300346d84/podcast.rss" Scrubs)))

(require 'elfeed-org)
(require 'elfeed-goodies)
(elfeed-goodies/setup)

(global-set-key (kbd "C-x w") 'elfeed)
(elfeed-org)
(setq rmh-elfeed-org-files (list "~/.emacs.d/elfeed-dashboard.org"))
```

**Comment by eshrh** [comment]
https://elpa.gnu.org/packages/avy.html

is a more generic solution for this kind of usecase, I'm not sure if we should be reimplementing significant logic in a probably worse way here.

**Reply by chookity-pokk** [reply]
Yeah, that seems like the way to go. Thanks for the link!
